.pre-commit-config.yaml

```diff
@@ -17,4 +17,4 @@ repos:
       - id: ruff-format
 
 ci:
-    autoupdate_schedule: monthly
\ No newline at end of file
+  autoupdate_schedule: monthly
```

Comment:
would this setting not have worked with the existing indent?

Reply:
oh it would, i didn't notice it changed... that's not from the eslint/prettier stuff, that was just me doing my habitual formatting hotkey spam. 
we can totally revert it